packages/backend/src/app.ts

```diff
@@ -611,8 +611,8 @@ export class App {
     if (this.openPaymentsServer) {
       await this.stopServer(this.openPaymentsServer)
     }
-    if (this.adminServer) {
-      await this.stopServer(this.adminServer)
+    if (this.apolloServer) {
+      await this.apolloServer.stop()
     }
```

Comment:
since we do `plugins: [ApolloServerPluginDrainHttpServer({ httpServer })]` when creating the apollo server, the httpServer will be automatically stopped/connections will be drained. However, before we weren't closing the actual apollo server that does other operations as well: https://www.apollographql.com/docs/apollo-server/api/apollo-server/#stop